views/sign_up.erb

```diff
@@ -0,0 +1,50 @@
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+
+<style>
+  body {
+    font-family: 'Quicksand', sans-serif;
+    text-align: center;
+    background: #99e6ff;
+  }
+
+  .inline {
+   display: inline-block;
+  }
+
+  input[type=text] {
+    border: 2px solid #000;
+    border-radius: 10px;
+    width: 300px;
+    height: 30px;
+  }
+
+  .button {
+    border: solid 2px;
+    border-radius: 15px;
+    color: #000000;
+    background-color: #FFFFFF;
+    padding: 1px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 20px;
+    font-weight: 50;
+    width: 100px;
+    font-weight: bold;
+  }
+</style>
+
+<form action='/new_user' method="post">
+  <label for="name">Name:</label>
+  <input type="text" name="name"><br><br>
+  <label for="user_handle">User Handle:</label>
+  <input type="text" name="handle" placeholder="@"><br><br>
+  <label for="email">Email Address:</label>
+  <input type="text" name="email"><br><br>
+  <label for="password">Password:</label>
+  <input type="text" name="password"><br><br>
```

Comment:
It would be best if you had the password as type="password" so that the password is not visible as you inputting it.